.gitignore

```diff
@@ -1,10 +1,71 @@
-node_modules
-package-lock.json
+.DS_Store
+dist/
+coverage/
+docs/
+
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
+
+# Diagnostic reports (https://nodejs.org/api/report.html)
+report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
+
```

Comment:
Nit: but if you want to ignore IDE generated files you can add these lines (these are included in `metamask-extension` too):
```
# IDEs
.idea
.sublime-project
```